### ISSUE TITLE
Many sites spamming

### ISSUE THREAD

**Issue by tf7software** [sherlock-project/sherlock#2148]
### Site name

A lot

### Additional info

There is APClips, Heavy-R, and EyeEm and more that say there are users for every name, please remove

**Comment by ppfeister** [comment]
EyeEm is fixed.

Heavy-R and APC are unable to be reproduced. Additional info such as region, if it worked until sherlock was ran x times, etc, may be helpful.

**Comment by tf7software** [comment]
ok, i also noticed:



CyberDefenders
RocketTube
Zhihu
Twitch (Sometimes)
hunting
igromania

**Comment by tf7software** [comment]
> EyeEm is fixed.
> 
> Heavy-R and APC are unable to be reproduced. Additional info such as region, if it worked until sherlock was ran x times, etc, may be helpful. Sometimes rate limiting, regional differences, etc, can play a role.

So, it only happens with the --nswf flag (Or ```Show NSFW Results``` On Watson), I am in the US, and I'll link a picture

**Comment by tf7software** [comment]
![WatsonScreenshot](https://i.im.ge/2024/06/02/K52Je1.Screenshot-2024-06-02-at-10-38-54AM.png)

**Comment by tf7software** [comment]
https://im.ge/i/Screenshot-2024-06-02-at-10-38-54AM.K52Je1

**Comment by ppfeister** [comment]
Only able to repro CyberDefenders and Zhihu over here (also US).

I've been considering adding a devel use flag for these situations, to more easily dump what Sherlock is actually seeing. I'll have to look more into doing that and ping you once it's added. Pretty good feeling it's rate limit related.

**Comment by tf7software** [comment]
Yeah, that would be nice, I could incorporate this into [Watson](https://github.com/tf7software/Watson), also do you think you guys could collab with me and have official support for Watson?

**Comment by tf7software** [comment]
Also, you can find my personal email [HERE](https://spamty.eu/show/v7/205/dcdbbabb07/)

**Comment by aytvill** [comment]
I just run my nickname for the first time with --nsfw flag, and it has been interesting, including that both APC and Heavy-R gave false positives

1. APC redirects on their landing /model and my profile obviously doesn't exist. When I execute `curl -I https://apclips.com/aytvill` it gives back this

>  curl -I https://apclips.com/aytvill
HTTP/2 403 
date: Thu, 21 Nov 2024 10:16:15 GMT
content-type: text/html; charset=UTF-8
x-frame-options: SAMEORIGIN
referrer-policy: same-origin
cache-control: max-age=15
expires: Thu, 21 Nov 2024 10:16:30 GMT
report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=5Y2CEF8nig7rNIS1jyliYIFjidNkMNx2E19%2Fent5bo0H1rBo9N8df%2Fw42eIpC%2FGbGVYJ6dDQbJUCPmxhc9xEp1DIbQbwKBVCrpAeY3v%2FfPOfA9R2fQZKXj1dk3GkPw%3D%3D"}],"group":"cf-nel","max_age":604800}
nel: {"success_fraction":0,"report_to":"cf-nel","max_age":604800}
server: cloudflare
cf-ray: 8e5fec5b4bab8d6d-HEL
alt-svc: h3=":443"; ma=86400
server-timing: cfL4;desc="?proto=TCP&rtt=15025&sent=5&recv=7&lost=0&retrans=0&sent_bytes=3410&recv_bytes=798&delivery_rate=168215&cwnd=251&unsent_bytes=0&cid=bc695b3aa47466df&ts=36&x=0"


2. Heavy-R shows small pink bar with 'Channel not found', but it does NOT redirect anywhere. When I execute `curl -I https://www.heavy-r.com/user/aytvill`  it gives back this

> $ curl -I https://www.heavy-r.com/user/aytvill
HTTP/2 403 
date: Thu, 21 Nov 2024 10:16:06 GMT
content-type: text/html; charset=UTF-8
x-frame-options: SAMEORIGIN
referrer-policy: same-origin
cache-control: max-age=15
expires: Thu, 21 Nov 2024 10:16:21 GMT
report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=P63XQSJMu%2FCjnCbqopB%2FguRhzUhsU3%2FZzCfQVft1fDDBtQppwE8fpRGlleCg955tXiP%2BmgcTgS5h5vfxmulX6B5vk1pTFViRdgYgxXdfpJS%2BK3YYPkzfp96EXYLuTk3MMg%3D%3D"}],"group":"cf-nel","max_age":604800}
nel: {"success_fraction":0,"report_to":"cf-nel","max_age":604800}
server: cloudflare
cf-ray: 8e5fec22983f8dc2-HEL
alt-svc: h3=":443"; ma=86400
server-timing: cfL4;desc="?proto=TCP&rtt=17943&sent=6&recv=8&lost=0&retrans=0&sent_bytes=3423&recv_bytes=837&delivery_rate=197733&cwnd=224&unsent_bytes=0&cid=d6bb5e44648754e2&ts=45&x=0"


**to sum it up**, both sites are questionable in their profile reporting, but issue 403 instead of 404 on non-existent profile
